src/pacow/cowsay.clj

```diff
@@ -2,8 +2,22 @@
   (:require [clojure.java.shell :refer [sh]]
             [environ.core :refer [env]]))
 
-(def bin (env :cowsay-bin "cowsay"))
+(def cowsay (env :cowsay-bin "cowsay"))
 
-(defn moo [m]
-  (let [out (:out (sh bin m))]
-    (str "```" out "```")))
+(defn slack-out [text]
```

Comment:
I think this function should probably belong in `slack.clj` since the markdown formatting is part of Slack, not cowsay. It was my mistake ever putting it in the `cowsay` namespace. Can you do that as part of this PR?